src/clj/game/core/def_helpers.clj

```diff
@@ -179,6 +179,18 @@
        (make-current-event-handler title)
        (make-recurring-ability)))
 
+(defn corp-recur
+  ([] (corp-recur (constantly true)))
+  ([pred]
+   {:label "add card from Archives to HQ"
+    :prompt "Select a card to add to HQ"
+    :show-discard true
+    :choices {:card #(and (corp? %)
+                       (in-discard? %)
+                       (pred %))}
+    :msg (msg "add " (if (faceup? target) (:title target) "an unseen card") " to HQ")
```

Comment:
Wanna try something for me? Change the `(if (faceup? ...` part of the msg to instead use `card-str` and see how that reads. If it reads like shit, leave this as is.

Reply:
Switched it over. Reads OK to me.